### PR TITLE
`sage.schemes.generic`: fix docs

### PR DESCRIPTION
**Pull request by grhkm21**
Fix various docs problems.

I also removed the `SchemeRationalPoint` class entirely since it is not used anywhere in the entire Sage library. If anyone oppose this please notify me!

### CODE REVIEW

File: src/sage/schemes/generic/glue.py [hunk 1]
```diff
@@ -16,19 +16,35 @@ class GluedScheme(scheme.Scheme):
     INPUT:
 
 
-    -  ``f`` - open immersion from a scheme U to a scheme
-       X
+    -  ``f`` - open immersion from a scheme `U` to a scheme
+       `X`
 
-    -  ``g`` - open immersion from U to a scheme Y
+    -  ``g`` - open immersion from `U` to a scheme `Y`
```
Comment (kwankyu) [inline src/sage/schemes/generic/glue.py:23]: While you are here, please `- ``f`` --`. Note two hyphens.

File: src/sage/schemes/generic/glue.py [hunk 2]
```diff
@@ -16,19 +16,35 @@ class GluedScheme(scheme.Scheme):
...
-    OUTPUT: The scheme obtained by gluing X and Y along the open set
-    U.
+    OUTPUT: The scheme obtained by gluing `X` and `Y` along the open set
+    `U`.
 
     .. note::
 
        Checking that `f` and `g` are open
        immersions is not implemented.
+
```
Comment (kwankyu) [inline src/sage/schemes/generic/glue.py:32]: While you are here, `.. NOTE::`.

File: src/sage/schemes/generic/glue.py [hunk 3]
```diff
@@ -16,19 +16,35 @@ class GluedScheme(scheme.Scheme):
...
+        sage: phi_x = Rx.hom([xbar])
+        sage: phi_y = Ry.hom([ybar])
+        sage: Sx = Schemes()(phi_x)
+        sage: Sy = Schemes()(phi_y)
+        sage: Sx.glue_along_domains(Sy)
+        Scheme obtained by gluing X and Y along U, where
+          X: Spectrum of Univariate Polynomial Ring in x over Rational Field
+          Y: Spectrum of Univariate Polynomial Ring in y over Rational Field
+          U: Spectrum of Quotient of Multivariate Polynomial Ring in x, y over Rational Field by the ideal (x*y - 1)
     """
```
Comment (kwankyu) [inline src/sage/schemes/generic/glue.py:48]: Nice example!

File: src/sage/schemes/generic/glue.py [hunk 4]
```diff
@@ -42,6 +58,21 @@ def __init__(self, f, g, check=True):
         self.__g = g
 
     def gluing_maps(self):
+        r"""
+        Return the gluing maps of this glued scheme, i.e. the maps `f` and `g`.
+
+        EXAMPLES::
+            sage: S.<xbar, ybar> = R.quotient(x * y - 1)
+            sage: Rx = QQ["x"]
```
Comment (kwankyu) [inline src/sage/schemes/generic/glue.py:68]: A blank line below `EXAMPLES::`.

**Comment by kwankyu** [comment]
> I also removed the `SchemeRationalPoint` class entirely since it is not used anywhere in the entire Sage library. If anyone oppose this please notify me!

I agree. `sage.schemes.generic.morphism.SchemeMorphism_point` would suffice for points as morphism.

**Comment by kwankyu** [comment]
Otherwise lgtm.

**Comment by grhkm21** [comment]
Thanks! I also find *a lot* of violation for the two edits you suggested, I'll make another PR just for them.

**Comment by grhkm21** [comment]
Sorry, the doctests are written wrong I think... let me take a look

**Comment by grhkm21** [comment]
@kwankyu Do you mind re-reviewing and accepting? It's a one line change :) Thanks!

**Comment by kwankyu** [comment]
Thanks. Tests pass on my system.

**Pull request merged**